src/components/PortfolioItem.jsx

```diff
@@ -69,7 +79,8 @@ export const PortfolioItem = ({ setPortfolioListNeedsUpdate }) => {
                     <CoinTag src={item.coinData.image.large} />
                   </div>
                   &nbsp;&nbsp;
-                  <div className="text-xl font-semibold text-skin-portfolio-item-coin-name-total-value-current-price-text-color">
+                  <div className="text-xl font-semibold text-skin-portfolio-item-coin-name-total-value-current-price-text-color" onClick={()=>{handleClick(item.coinData)}}>
```

Comment:
do you need to wrap handleClick with {}. I would do {() => handleClick(arg)}. 
I would only add () => { 
// some logic
return {}
}

In case the  if logic would exceed one line.